### ISSUE TITLE
SCSS @include indentation

### ISSUE THREAD

**Issue by TMcLoone** [Glavin001/atom-beautify#994]
Sorry if this has been included somewhere else but I noticed that when nesting an @include within an @include, the second include has the wrong indentation. 

So, for example it should look something like this: 

```
.example {
    @include media($desktop-min) {
        @include span-columns(4);
        padding: 1em;
    }
}
```

But it ends up like this:

```
.example {
    @include media($desktop-min) {
         @include span-columns(4);
        padding: 1em;
    }
}
```

**Comment by prettydiff** [comment]
I have verified this is a Pretty Diff defect and I have opened an issue to track this.

**Comment by tomasbedrich** [comment]
This bug also affects `@include` inside a `@mixin`:

```
@mixin customer-area-base-box {
   @include box-shadow;
  background: white;
  padding: rem(16px);
}
```

So I think the `@` character can cause this problem.

**Comment by prettydiff** [comment]
The problem is that Atom Beautify is using the latest version of Pretty Diff available from NPM (v1.16.37) from late April.  I have since fixed this problem around 5 June.  I am working on an alternative to NPM and promised to dedicate time to working through my current technical hurdle this weekend.

**Comment by UziTech** [comment]
Any update on this?

A simple fix is to add a comment between the two lines

```
@mixin customer-area-base-box {
  // comment to prevent https://github.com/Glavin001/atom-beautify/issues/994
  @include box-shadow;
  background: white;
  padding: rem(16px);
}
```

**Comment by prettydiff** [comment]
Yes, my alternative to NPM is almost ready: https://github.com/prettydiff/biddle

Current status:

The application is currently in beta mode.  On Monday night I successfully executed it against Pretty Diff on its production domain.  I have written a test suite for it which is passing on multiple operating systems.  Everything looks good to go for public launch.  I am working on a last initial feature right now before I launch this into the wild.  I hope to have this last feature complete ASAP so that I can promote biddle on prettydiff.com to encourage additional use and testing.

Be aware though.  The biddle application is working for me in OSX, Windows, Semaphore, and AppVeyor.  Nobody else has tested this yet, so really it could still be buggy as hell.  If you would like to help test just clone biddle down and go through the "Getting Started" guide in the documentation.  The documentation is all in markdown format so you can easily read it on Github or from the terminal since biddle includes a markdown parser.

I hope the biddle project will be very helpful to this project.  Eventually Atom-Beautify will be replaced by Unibeautify.  NPM is not enough to solve that distribution problem, because NPM is for Node related code where biddle will run anywhere Node runs and support any language/platform/run time.

**Comment by prettydiff** [comment]
Initial Pretty Diff publication via biddle is online in production: http://prettydiff.com/downloads/prettydiff/

Once you download biddle its as easy as:

    node biddle global
    biddle install http://prettydiff.com/downloads/prettydiff/prettydiff_cli_latest.zip

I will work on the next patch of Pretty Diff now in order to make biddle support formal to Pretty Diff's master branch and also fix some outstanding bugs.  This will be part of an on-going beta process to work out defects and try to test features of the application in a way where these features are exposed to end users in production.

**Comment by prettydiff** [comment]
Keeping everybody informed I have fixed biddle.  Check it out now - https://github.com/prettydiff/biddle

I will republish Pretty Diff using biddle in production tomorrow.  I am not ready to give it to Atom Beautify just yet though.  I actually need users to play around with this thing for a bit so that I can get a sense of where the pain points are and just how bad it works for fails for other people.  I wrote a getting started document at https://github.com/prettydiff/biddle/blob/master/documentation/gettingstarted.md

The documentation might be easier to read from the command line though as I included a markdown to CLI parser in biddle.  The number at the end defines the word wrap limit:

    node biddle markdown documentation/gettingstarted.md 80
